src/main/scala/com/beachape/filemanagement/FileSystemWatchMessageForwardingActor.scala

```diff
@@ -0,0 +1,59 @@
+package com.beachape
+
+/**
+ * Created by arunavs on 9/27/16.
+ *
+ * This is a simple actor that can be used directly to send a message
+ * whenever a dir is modified.
+ *
+ * It sends the changed dir message whenever changes happen to the
+ * directory.
+ */
+
+import java.nio.file.Paths
+import java.nio.file.StandardWatchEventKinds._
+
+import akka.actor.ActorRef
+import com.beachape.filemanagement.Messages.{ EventAtPath, RegisterCallback }
+import com.beachape.filemanagement.MonitorActor
+import com.beachape.filemanagement.RegistryTypes.Callback
+
+class FileSystemWatchMessageForwardingActor(dirs: Seq[String],
```

Comment:
Instead of a `Seq[String]`, I'd prefer this to be a `Seq[Path]` or something like that so we don't have a stringly-typed API.